Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         autoconf \
         m4 \
         gperf && \
-    pip install ply && \
+    pip3 install ply && \
```

Comment:
The above installed packages should also be changed to their `python3-*` variants.

Reply:
I think the same, @rcombs can you change to python3-ply?